core/pom.xml

```diff
@@ -23,12 +23,35 @@ limitations under the License.
     <version>1.22.0-SNAPSHOT</version>
   </parent>
 
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
   <artifactId>calcite-core</artifactId>
   <packaging>jar</packaging>
   <version>1.22.0-SNAPSHOT</version>
   <name>Calcite Core</name>
   <description>Core Calcite APIs and engine.</description>
 
+  <developers>
+    <developer>
+      <name>Apache Calcite developers list</name>
+      <email>dev@calcite.apache.org</email>
+      <organization>Apache Calcite</organization>
+      <organizationUrl>https://calcite.apache.org</organizationUrl>
+    </developer>
+  </developers>
```

Comment:
Should this reflect LinkedIn or Apache Calcite? Is there an equivalent section in upstream Calcite?

Reply:
good point, I copied from root pom.xml  <mailingList>. What is the LinkedIn calcite dev email?

Reply:
If it is copied from root, then I think it is fine, but I do not see it in the root pom.xml. Could you please share a pointer?

Reply:
https://github.com/linkedin/linkedin-calcite/blob/931e80e6f1d08733b98e71719372eea7177b735a/pom.xml#L46

Reply:
Thinking again, I think it is fair for this to be LinkedIn-specific since there are LinkedIn specific patches in this module. In that case, we can list devs of this repo.

Reply:
We can add add a LinkedIn-specific section after this section, as another entry.